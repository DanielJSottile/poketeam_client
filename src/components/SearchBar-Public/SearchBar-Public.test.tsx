import React from 'react';
import ReactDOM from 'react-dom';
import { BrowserRouter } from 'react-router-dom';
//import { render } from '@testing-library/react';
import SearchBarPublic from './SearchBar-Public';
import { shallow, mount, render } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('<SearchBarPublic/>', (): void => {
  it('renders without crashing', (): void => {
    const div: HTMLDivElement = document.createElement('div');
    ReactDOM.render(
      <BrowserRouter>
        <SearchBarPublic/>
      </BrowserRouter>
      , div);
      ReactDOM.unmountComponentAtNode(div);
  });
  it('snapshot', (): void => {
    const wrapper = shallow(<SearchBarPublic/>);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});