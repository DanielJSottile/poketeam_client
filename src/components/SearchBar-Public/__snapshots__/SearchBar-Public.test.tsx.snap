// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchBarPublic/> snapshot 1`] = `
<div
  className="searchbar"
>
  <form
    className="search"
  >
    <div
      className="search"
    >
      <label
        htmlFor="search"
      >
        Search: 
        <i
          className="fas fa-search"
        />
      </label>
      <input
        className="s-input"
        id="search"
        name="search"
        onChange={[Function]}
        placeholder="e.g. Pikachu"
        type="text"
        value=""
      />
      <label
        htmlFor="sort"
      >
        Sort By:
        <i
          className="fas fa-sort"
        />
         
      </label>
      <select
        id="sort"
        name="sort"
        onChange={[Function]}
        value=""
      >
        <option
          value="newest"
        >
          Newest
        </option>
        <option
          value="oldest"
        >
          Oldest
        </option>
        <option
          value="alphabetical"
        >
          A - Z
        </option>
        <option
          value="rev alphabetical"
        >
          Z - A
        </option>
      </select>
      <button
        onClick={[Function]}
        type="submit"
      >
        Go 
        <i
          className="fas fa-exclamation"
        />
      </button>
    </div>
  </form>
</div>
`;
